### PR TITLE
features

### PR DESCRIPTION
**Pull request by rbaumier**

### CODE REVIEW

File: src/views/features.jade [hunk 1]
```diff
@@ -52,13 +86,30 @@ block content
...
+                  .title-card Auto-refresh
+                  .description-card Visualize your data in real-time.
+
+          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
+            .card-container
+              .feature-card
+                img.screen(src='im/features/console-card.jpg', alt='')
+                .text-container
+                  .title-card Browser API
+                  .description-card Embedded Redis JavaScript API. NodeJS node_redis API directly from your console.
```
Comment (FGRibreau) [inline src/views/features.jade]: "NodeJS node_redis API directly from your console." -> "Test your JavaScript against Redis directly from your browser console."

File: src/views/features.jade [hunk 2]
```diff
@@ -52,13 +86,30 @@ block content
...
 
-          .col-xs-4
+        .row.feature-row
+          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
+            .card-container
+              .feature-card
+                img.screen(src='im/features/autorefresh-card.jpg', alt='')
+                .text-container
+                  .title-card Auto-refresh
+                  .description-card Visualize your data in real-time.
```
Comment (FGRibreau) [inline src/views/features.jade]: >Do yo use Redis for real-time analytics? Redsmin helps you visualize it in real-time! 

?

File: src/views/features.jade [hunk 3]
```diff
@@ -35,16 +53,32 @@ block content
...
+                  .description-card Plot everything, visualize hash or a zset updated in real-time.
 
-          .col-xs-4
+          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
+            .card-container
+              .feature-card
+                img.screen(src='im/features/createinstance-card.jpg', alt='')
+                .text-container
+                  .title-card Proxy
+                  .description-card Connect Redsmin to both direct and indirect/proxied instances.
```
Comment (FGRibreau) [inline src/views/features.jade]: > Connects to everything, from cloud, to your own server to locally hosted Redis instances.

File: src/views/features.jade [hunk 4]
```diff
@@ -35,16 +53,32 @@ block content
...
+          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
             .card-container
               .feature-card
-                img.screen(src='im/features/editor-card.jpg', alt='')
+                img.screen(src='im/features/graph-card.jpg', alt='')
                 .text-container
-                  .title-card Control your data
-                  .description-card Manage your data with our powerful editor supporting key batch operations.
+                  .title-card Keep control
+                  .description-card Plot everything, visualize hash or a zset updated in real-time.
```
Comment (FGRibreau) [inline src/views/features.jade]: >Do yo use Redis for real-time analytics? Redsmin helps you visualize it in real-time! 

?

File: src/views/features.jade [hunk 5]
```diff
@@ -35,16 +53,32 @@ block content
                   .title-card Advanced Key Finder
                   .description-card Look up into your database with thousands of keys through patterns.
 
+          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
+            .card-container
+              .feature-card
+                img.screen(src='im/features/config-card.jpg', alt='')
+                .text-container
+                  .title-card Arrange at your will
+                  .description-card Configure your Redis instance directly from the GUI.
```
Comment (FGRibreau) [inline src/views/features.jade]: > Redsmin will help you configure your Redis instance with inline-documentation

?

File: src/views/features.jade [hunk 6]
```diff
@@ -19,15 +27,25 @@ block content
...
-          .col-xs-4
+
+        .row.feature-row
+          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
+            .card-container
+              .feature-card
+                img.screen(src='im/features/finder-card.jpg', alt='')
+                .text-container
+                  .title-card Key batch operations
+                  .description-card Ever wanted to edit a zset or a hash field? With Redsmin you can.
```
Comment (FGRibreau) [inline src/views/features.jade]: ->  Key batch operations -> advanced data editor

Comment (rbaumier) [inline src/views/features.jade]: "Modern data editor" ? On a déjà "Advanced Key Finder" :)

File: src/views/features.jade (hunk 3 above)
Comment (rbaumier) [inline src/views/features.jade]: :+1:  Je mets le "Connects to everything" en titre pour que ça rentre en 2 lignes :)

Comment (FGRibreau) [inline src/views/features.jade]: Ok !

Le jeudi 16 juillet 2015, Romain Baumier <notifications@github.com> a
écrit :

> In src/views/features.jade
> <https://github.com/Redsmin/website/pull/60#discussion_r34786382>:
>
> >
> > -          .col-xs-4
> > +          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
> > +            .card-container
> > +              .feature-card
> > +                img.screen(src='im/features/createinstance-card.jpg', alt='')
> > +                .text-container
> > +                  .title-card Proxy
> > +                  .description-card Connect Redsmin to both direct and indirect/proxied instances.
>
> [image: :+1:] Je mets le "Connects to everything" en titre pour que ça
> rentre en 2 lignes :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Redsmin/website/pull/60/files#r34786382>.
>


-- 
Sent from my iPhone

File: src/views/features.jade (hunk 6 above)
Comment (FGRibreau) [inline src/views/features.jade]: Let's try this!

Le jeudi 16 juillet 2015, Romain Baumier <notifications@github.com> a
écrit :

> In src/views/features.jade
> <https://github.com/Redsmin/website/pull/60#discussion_r34785718>:
>
> >                    .description-card Almost too simple to use with integrated autocomplete and inline documentation.
> >
> > -          .col-xs-4
> > +
> > +        .row.feature-row
> > +          .col-xs-4(data-center="transform: translateY(0px); opacity: 1;" data--100-bottom="transform: translateY(50px); opacity: 0.5;")
> > +            .card-container
> > +              .feature-card
> > +                img.screen(src='im/features/finder-card.jpg', alt='')
> > +                .text-container
> > +                  .title-card Key batch operations
> > +                  .description-card Ever wanted to edit a zset or a hash field? With Redsmin you can.
>
> "Modern data editor" ? On a déjà "Advanced Key Finder" :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Redsmin/website/pull/60/files#r34785718>.
>


-- 
Sent from my iPhone

File: src/views/features.jade (hunk 4 above)
Comment (rbaumier) [inline src/views/features.jade]: "real-time" -> "immediatly" ? On a une répétition de "real-time" sinon, surtout qu'on le dit pas mal dans les autres cards ^^

File: src/public/js/main.js [hunk 7]
```diff
@@ -71,12 +72,16 @@ $(function () {
...
   });
 
-  function scrollTop()  {
+  function scrollTop() {
     $('body').animate({
       scrollTop: 0
     }, 800);
   }
 
+  if (location.href.indexOf('create_account') !== -1) {
```
Comment (FGRibreau) [inline src/public/js/main.js]: c'est très sale, on va modifier ça TRES rapidement :)

**Pull request merged**